packages/paste-core/components/alert-dialog/src/index.tsx

```diff
@@ -1,108 +1,4 @@
-  enter: {opacity: 1, transform: `scale(1)`},
-  leave: {opacity: 0, transform: `scale(0.675)`},
-  // https://www.react-spring.io/docs/hooks/api
-  config: {
-    mass: 0.5,
-    tension: 370,
-    friction: 26,
-  },
-});
-
-export interface AlertDialogProps extends React.HTMLAttributes<HTMLElement> {
-  children: React.ReactNode;
-  destructive?: boolean;
-  heading: string;
-  isOpen: boolean;
-  onConfirm: () => void;
-  onConfirmLabel: string;
-  onDismiss: () => void;
-  onDismissLabel: string;
-}
-
-export const AlertDialog = React.forwardRef<HTMLDivElement, AlertDialogProps>(
-  ({children, destructive, heading, isOpen, onConfirm, onConfirmLabel, onDismiss, onDismissLabel, ...props}, ref) => {
-    const transitions = useTransition(isOpen, getAnimationStates());
-    const headingID = useUID();
-    const bodyID = useUID();
-
-    const primaryVariant = destructive ? 'destructive' : 'primary';
-
-    return (
-      <>
-        {transitions(
-          (styles, item) =>
-            item && (
-              <ModalDialogOverlay isOpen={isOpen} style={{opacity: styles.opacity}}>
-                <NarrowModalDialogContent
-                  {...safelySpreadBoxProps(props)}
-                  aria-labelledby={headingID}
-                  aria-describedby={bodyID}
-                  ref={ref}
-                  role="alertdialog"
-                  style={styles}
-                >
-                  <Box as="header" {...modalHeaderStyles} border="none" paddingBottom="space0">
-                    <Heading as="h3" marginBottom="space0" variant="heading30" id={headingID}>
-                      {heading}
-                    </Heading>
-                  </Box>
-                  <Box as="div" {...modalBodyStyles} id={bodyID}>
-                    {children}
-                  </Box>
-                  <Box as="footer" {...modalFooterStyles} border="none" justifyContent="flex-end" paddingTop="space0">
-                    <Stack orientation="horizontal" spacing="space50">
-                      <Button variant="secondary" onClick={onDismiss}>
-                        {onDismissLabel}
-                      </Button>
-                      <Button variant={primaryVariant} onClick={onConfirm}>
-                        {onConfirmLabel}
-                      </Button>
-                    </Stack>
-                  </Box>
-                </NarrowModalDialogContent>
-              </ModalDialogOverlay>
-            )
-        )}
-      </>
-    );
-  }
-);
-
-AlertDialog.displayName = 'AlertDialog';
-
-AlertDialog.propTypes = {
-  children: PropTypes.node.isRequired,
-  destructive: PropTypes.bool,
-  heading: PropTypes.string.isRequired,
-  isOpen: PropTypes.bool.isRequired,
-  onConfirm: PropTypes.func.isRequired,
-  onConfirmLabel: PropTypes.string.isRequired,
-  onDismiss: PropTypes.func.isRequired,
-  onDismissLabel: PropTypes.string.isRequired,
-};
+export * from './AlertDialog';
+export * from './AlertDialogHeader';
```

Comment:
Actually, lets just expose AlertDialog from the package. The other bits can be private as you directly import them from source in stories and tests.

That way we're not confusing consumers with additional components that aren't actually part of the documented API

Reply:
Good call. I'll make that change.